tools/lxdclient/config.go

```diff
@@ -60,13 +60,24 @@ func (cfg Config) UsingTCPRemote() (Config, error) {
 		return cfg, nil
 	}
 
+	/* Here, we rely on the fact that Connect() does a ServerStatus.
```

Comment:
Oh, I see what's going on here. We don't do it for HTTP connections, but this is always a unix connection since we enforce it just above. I can do an explicit ServerStatus to guard against this changing if you want, but I think we always have to do it on the unix side in the LXD client code because we need to download the server's cert.